environment.yml

```diff
@@ -22,6 +22,8 @@ dependencies:
     - pygeobase
     - ascat
     - ismn
+    - repurpose
```

Comment:
repurpose is now double

Reply:
Also add the minimum version number that is necessary. Otherwise older versions will be accepted which will break

Reply:
Just seen it in requirements. So scratch the last comment